src/main/java/edu/cornell/kfs/module/purap/CUPurapKeyConstants.java

```diff
@@ -23,15 +23,16 @@ public class CUPurapKeyConstants {
     public static final String ERROR_EXCEED_SQ_NUMBER_OF_ATT_LIMIT = "errors.exceed.sq.number.of.att.limit";
     public static final String ERROR_REASON_IS_REQUIRED = "errors.reason.is.required";
     public static final String ERROR_ATT_FILE_SIZE_OVER_LIMIT = "errors.att.filesize.over.limit";
-   
+    public static final String ERROR_ATT_SEND_TO_VENDOR_FILE_SIZE_OVER_LIMIT = "errors.att.send.to.vendor.filesize.over.limit";

```

Comment:
I see you created a new property key to replace the old `errors.att.filesize.over.limit` property. I think the old property key was only used for this custom filesize check, so if you're not using it then you may want to remove it (or create a follow-up ticket to do so).